# type of object
string object_type

# movement (position, heading, speed, acceleration) of the perceived object
ObjectMovement object_movement

# length of the object in x direction
float64 x_length

# the length of the object in y direction
float64 y_length